pipelines/crawler/bndes/constants.py

```
@@ -0,0 +1,148 @@
+        "https://dadosabertos.bndes.gov.br/api/3/action/resource_show"
+        "?id=612faa0b-b6be-4b2c-9317-da5dc2c0b901"
+    )
+    DOWNLOAD_URL = (
+        "https://dadosabertos.bndes.gov.br/dataset/"
+        "10e21ad1-568e-45e5-a8af-43f2c05ef1a2/resource/"
+        "612faa0b-b6be-4b2c-9317-da5dc2c0b901/download/"
+        "operacoes-financiamento-operacoes-indiretas-automaticas.csv"
+    )
+    LAST_MODIFIED_FORMAT = "%Y-%m-%dT%H:%M:%S.%f"
+
+    INPUT_PATH = "/tmp/input/br_bndes_operacoes_contratadas/"
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/input/br_bndes_operacoes_contratadas/"

(S108)